### ISSUE TITLE
Add support for custom properties

### ISSUE THREAD

**Issue by colinmeinke** [atom/language-css#48]
As a syntax theme author I would love support for [CSS custom properties](http://www.w3.org/TR/css-variables/) which allows the following syntax in CSS:

``` css
:root {
  --bg-color: rgb( 255, 255, 255 );
}

body {
  background-color: var( --bg-color );
}
```

It would be great to get the variable class for `--bg-color` so I can style consistently between CSS, Sass and Less.

**Comment by 50Wliu** [comment]
Fixed by #99.
Thanks for clearing out the issue list @Alhadis.

**Comment by Alhadis** [comment]
I'd say #51 can be closed too, because with the exception of nested selectors, there's seriously nothing else to be done.
